docs/DesignDocs/AttestationApiRequirements.md

```diff
@@ -0,0 +1,291 @@
+Proposed Attestation API Requirements
+=====================================
+
+This design document proposes requirements for APIs that applications can
+use for performing attestation of enclave applications.
+
+Motivation
+----------
+
+Requirements:
+* Usable by Attesters, Verifiers, and Relying Parties
+* Able to support different TEEs
+* Able to support different protocols
+* Able to support different topological models
+  (e.g., Passport model vs Background-check model)
+* Able to support different Evidence formats
+* Able to support different Attestation Results formats
+* Able to support different Verifier services, including a local Verifier
+  on the same device as an Attester and/or a Relying Party
+* Able to support Verifiers and Relying Parties that might or might not run
```

Comment:
There are Verifiers and Relying Parties that are implemented in languages other than C/C++ and do not use the OE SDK at all.

Reply:
True, but not directly applicable to these requirements.  That's the job of the protocol definition which is covered by line 13